pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -653,6 +654,10 @@ func (cm *containerManagerImpl) GetPluginRegistrationHandler() cache.PluginHandl
 	return cm.deviceManager.GetWatcherHandler()
 }
 
+func (cm *containerManagerImpl) GetPluginRegistrationServerChecker() healthz.HealthChecker {
```

Comment:
please return the set of checkers so in future we may add more checkers, for example, for DRA